masonry_core/src/passes/event.rs

```diff
@@ -361,6 +361,14 @@ pub(crate) fn run_on_access_event_pass(
                 handled = Handled::Yes;
             }
         }
+        accesskit::Action::ScrollIntoView if !handled.is_handled() => {
+            let widget_state = root.widget_arena.get_state(target);
+            let rect = widget_state.item.layout_rect();
+            root.global_state
+                .scroll_request_targets
+                .push((target, rect));
+            handled = Handled::Yes;
+        }
```

Comment:
Indeed - if we land this, we can remove the equivalent code in #1077 (see https://github.com/linebender/xilem/pull/1077#discussion_r2219635181).